src/main/java/com/example/app/controller/UserController.java

```diff
@@ -0,0 +1,32 @@
+package com.example.app.controller;
+
+import com.example.app.entity.User;
+import com.example.app.service.impl.UserServiceImpl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
```

Comment:
Let's change import ordering to maintain the consistency of the entire project.
I suggest you place the imports in this order: java -> javax -> <other imports> -> project imports.